public/app.js

```diff
@@ -216,5 +217,36 @@
         });
 
         $httpProvider.interceptors.push('jwtInterceptor');
-    }
+
+        $translateProvider
+            .useSanitizeValueStrategy('sanitizeParameters')
```

Comment:
couldn't find a good description of what sanitize value strategy is all about... best I could find is [a StackOverflow answer](https://stackoverflow.com/questions/35090548/difference-between-angular-translates-usesanitizevaluestrategysanitize-vs/39118996)

Do you know whether any of the other strategies might be appropriate?

Reply:
https://angular-translate.github.io/docs/#/guide/19_security is the documentation for the sanitization strategy, it's definitely worth considering which strategy you want to go for, or experiment with. I think the minimum is to sanitize variables, but allow html in the translation strings for things like `<bold>`